tests/test_jupyter_notebook_nbconvert/env.yaml

```diff
@@ -0,0 +1,7 @@
+channels:
+  - conda-forge
+  - bioconda
+dependencies:
+  - python >=3.5
+  - python >=3.5
```

Comment:
The rabbit's suggestion was not correct 😅 This line is duplicated

Reply:
WHAT? In the display, I only saw the addition of an empty line! Ok, let's figure out how to revert that commit ...

Reply:
ok, the web interface does not allow reverting a commit and I cannot push to your branch, at is a personal fork. Mind reverting my commit?

Reply:
Done!